recipe/meta.yaml

```diff
@@ -18,12 +18,14 @@ build:
 
 requirements:
   build:
-    - toolchain
     - pkg-config
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
     - nasm
-    - zlib 1.2.11
+    - zlib
```

Comment:
Please specify the minimum version of zlib that comes with run_exports here.

Reply:
this is done by the config files generated by conda-smithy

Reply:
ok.